src/screens/Settings.js

```diff
@@ -0,0 +1,195 @@
+import React, {useState, useContext} from 'react';
+import {
+    Modal,
+    StyleSheet,
+    Text,
+    View,
+    TextInput,
+    TouchableOpacity,
+    FlatList,
+} from 'react-native';
+import {Picker} from '@react-native-picker/picker';
+import Icon from "@expo/vector-icons/AntDesign";
+import { ServiceContext } from "../store/ServiceContext";
+import globalStyles from "../css/global";
+import EmailRow from "../components/EmailRow";
+
+const Settings = ({modalVisible, hideModal}) => {
+    const [toEmailType, setToEmailType] = useState("to");
+    const [toEmail, setToEmail] = useState("");
+    const [emails, setEmails] = useState([
+        // sample default email
+        // TODO: add integration for fetching  settings/emails
+        {
```

Comment:
move to src/model/mock

Reply:
instead of model/mock insert this inside the model/fixture/settings.js (You have to create this directory on your own)